luaui/Widgets/api_resource_spot_builder.lua

```diff
@@ -1,24 +1,17 @@
 function widget:GetInfo()
 	return {
-		name = "API Resource Spot Builder",
+		name = "API Resource Spot Builder (mex/geo)",
 		desc = "Handles construction of metal extractors and geothermal power plants for other widgets",
-		author = "Google Frog, NTG (file handling), Chojin (metal map), Doo (multiple enhancements), Floris (mex placer/upgrader), Tarte (maintenance/geothermal)",
+		author = "Hobo Joe, badosu, Google Frog, NTG, Chojin, Doo, Floris, Tarte",
 		version = "2.0",
 		date = "Oct 23, 2010; last update: April 12, 2022",
 		license = "GNU GPL, v2 or later",
 		handler = true,
-		layer = 0,
-		enabled = true  --  loaded by default?
+		layer = -1,
```

Comment:
why -1?

Reply:
This was loading after `extractor_snap`, which relies on it, so I swapped their positions. I can change it to whatever, as long as the order of these various widgets remains the same.

Reply:
```suggestion
		layer = -1, -- before `extractor_snap` which relies on it
```